app/scripts/modules/ecs/src/serverGroup/details/events/events.controller.ts

```diff
@@ -0,0 +1,51 @@
+import { IController, module } from 'angular';
+import { IModalServiceInstance } from 'angular-ui-bootstrap';
+
+import { ServerGroupEventsReader } from './serverGroupEventsReader.service';
+import { IServerGroup } from '../../../../../core/src/domain/index';
+
+export interface IScalingActivitiesViewState {
+  loading: boolean;
+  error: boolean;
+}
+
+export interface IEventDescription {
+  createdAt: number;
+  message: string;
+  id: string;
+  status: string;
+}
+
+export class EventsController implements IController {
+  public viewState: IScalingActivitiesViewState;
+  public events: IEventDescription[] = [];
+
+  public constructor(private $uibModalInstance: IModalServiceInstance, public serverGroup: IServerGroup) {
+    'ngInject';
+    this.serverGroup = serverGroup;
```

Comment:
This is redudant; the inlined access modifiers in the constructor's parameters do exactly this.